### PR TITLE
file structure

### PR DESCRIPTION
**Pull request by RamyAlshurafa**

### CODE REVIEW

File: src/database/db_connect.js [hunk 1]
```diff
@@ -0,0 +1,26 @@
+const { Pool } = require('pg');
+
+const url = require('url');
+require('env2')('./config.env');
+
+let { DB_URL } = process.env;
+console.log(process.env.DB_DEV_URL);
```
Comment (mohanagy) [inline src/database/db_connect.js]: remove it

File: src/database/db_connect.js [hunk 2]
```diff
@@ -0,0 +1,26 @@
+const { Pool } = require('pg');
```
Comment (mohanagy) [inline src/database/db_connect.js]: i think we said that we gonna develop the project with sequalize

File: src/controllers/index.js [hunk 3]
```diff
@@ -0,0 +1,10 @@
+const express = require('express');
+const home = require('./home');
+const login = require('./login');
+
+const router = express.Router();
+
+router.get('/', home.get);
+router.get('/admin/login', login.get);
```
Comment (mohanagy) [inline src/controllers/index.js]: try to reduce  your code 
you can do this 
router
.get('/',home.get)
.get('/admin/login',login.get);

Comment (mohanagy) [inline src/controllers/index.js]: i think we said we gonna use RESTFUL API structure 
so we should follow it

File: src/database/build_db.sql [hunk 4]
```diff
@@ -0,0 +1,112 @@
...
+
+CREATE TABLE "rooms" (
+	"id" serial PRIMARY KEY ,
+	"room_name" varchar NOT NULL UNIQUE,
+	"capacity" integer NOT NULL,
+	"lcd" BOOLEAN NOT NULL
+) ;
+
+
+CREATE TABLE "events_courses" (
```
Comment (mohanagy) [inline src/database/build_db.sql]: are the events and courses same?

File: src/database/build_db.sql [hunk 5]
```diff
@@ -0,0 +1,112 @@
...
+	"id" serial PRIMARY KEY,
+	"title" varchar NOT NULL,
+	"description" TEXT NOT NULL,
+	"fees" integer
```
Comment (mohanagy) [inline src/database/build_db.sql]: i think it can be null right ?

File: src/database/build_db.sql [hunk 6]
```diff
@@ -0,0 +1,112 @@
...
+CREATE TABLE "staff" (
+	"id" serial PRIMARY KEY ,
+	"username" varchar NOT NULL UNIQUE,
+	"role" varchar NOT NULL,
+	"password" varchar NOT NULL
+);
+
+
+
+CREATE TABLE "users" (
```
Comment (mohanagy) [inline src/database/build_db.sql]: what are the users?
don't they have a password?
are the staff type of user?

File: src/database/build_db.sql [hunk 7]
```diff
@@ -0,0 +1,112 @@
...
+(3,1),
+(4,2),
+(1,2)
+
+;
+INSERT INTO bookings (event_id,start,"end",room_id) values
+(3, '2018-06-25 12:30:00', '2018-06-25 14:30:00',3),
+(4, '2018-06-28 16:00:00', '2018-06-28 18:30:00',2),
+(1, '2018-06-24 09:00:00', '2018-06-24 11:30:00',1),
+(NULL, '2018-06-26 10:00:00', '2018-06-26 12:30:00',1),
```
Comment (mohanagy) [inline src/database/build_db.sql]: why this value is null

File: src/controllers/index.js [hunk 8]
```diff
@@ -0,0 +1,10 @@
+const express = require('express');
```
Comment (mohanagy) [inline src/controllers/index.js]: i would love to see you start coding using ECMA6

File: src/database/build_db.js [hunk 9]
```diff
@@ -0,0 +1,12 @@
+const path = require('path');
+const fs = require('fs');
+
+const dbConnection = require('./db_connect');
+
+const sql = fs.readFileSync(path.join(__dirname, 'build_db.sql')).toString();
```
Comment (mohanagy) [inline src/database/build_db.js]: where are you calling this file?

File: src/controllers/index.js (hunk 8 above)
Comment (amusameh) [reply src/controllers/index.js]: What should we do here?

File: package.json [hunk 10]
```diff
@@ -0,0 +1,50 @@
...
+    "body-parser": "^1.18.2",
+    "cookie-parser": "^1.4.3",
+    "env2": "^2.2.0",
+    "express": "^4.16.3",
+    "express-handlebars": "^3.0.0",
+    "favicon": "0.0.2",
+    "jsonwebtoken": "^8.2.1",
+    "pg": "^7.4.3",
+    "pg-escape": "^0.2.0",
+    "pg-hstore": "^2.3.2",
```
[the remark this answers is not in the document]
Comment (amusameh) [reply package.json]: pg-hstore is required in the setup according to the documentation.
http://docs.sequelizejs.com/manual/installation/getting-started.html
What do you think?

File: src/database/models/clients.js [hunk 11]
```diff
@@ -0,0 +1,14 @@
+export default (sequelize, DataTypes) => {
+  const Clients = sequelize.define('clients', {
+    name: {
+      type: DataTypes.STRING,
+    },
+    phone: {
+      type: DataTypes.STRING,
+    },
+    email: {
```
[the remark this answers is not in the document]
Comment (amusameh) [reply src/database/models/clients.js]: edited :)

File: src/database/models/staff.js [hunk 12]
```diff
@@ -0,0 +1,22 @@
+import bcrypt from 'bcryptjs';
+
+export default (sequelize, DataTypes) => {
+  const Staff = sequelize.define('staff', {
+    name: {
```
[the remark this answers is not in the document]
Comment (amusameh) [reply src/database/models/staff.js]: It's a username
edited

File: package.json (hunk 10 above)
[the remark this answers is not in the document]
Comment (mohanagy) [reply package.json]: so we need it :D

File: src/database/models/staff.js (hunk 12 above)
[the remark this answers is not in the document]
Comment (mohanagy) [reply src/database/models/staff.js]: push

File: package.json [hunk 13]
```diff
@@ -0,0 +1,50 @@
...
+  "keywords": [],
+  "author": "",
+  "license": "ISC",
+  "bugs": {
+    "url": "https://github.com/FACG4/gmpLibrary/issues"
+  },
+  "homepage": "https://github.com/FACG4/gmpLibrary#readme",
+  "devDependencies": {
+    "babel-cli": "^6.26.0",
+    "babel-preset-stage-3": "^6.24.1",
```
[the remark this answers is not in the document]
Comment (amusameh) [reply package.json]: edited

**Pull request merged**